Dependencies/xr/Include/XR.h

```diff
@@ -223,6 +223,7 @@ namespace xr
                 void UpdateAnchor(Anchor&) const;
                 void DeleteAnchor(Anchor&) const;
                 Plane& GetPlaneByID(Plane::Identifier) const;
+                bool IsTracking() const;
```

Comment:
One thing we might want to consider setting IsTracking at frame creation time, and not having this query the  AR Session everytime it is called.  In general the frame object should just be a representation of state at the time the frame was created.